src/components/ErrorShow/ErrorHide.scss

```diff
@@ -0,0 +1,31 @@
+@keyframes hide {
```

Comment:
Consider creating a mixin for the repeated animation blocks to follow DRY (Don't Repeat Yourself) principle. 🌟